spec/features/unauthenticated_user_browse_by_category_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'unauthenticated user', type: :feature do
+  it 'can browse items by category' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.